### PR TITLE
Add Working Paper in motions

### PR DESCRIPTION
**Pull request by JulianWww**
Add working paper as described in #174. It just changes a couple of configuration files. I'm unsure if this should be procedural so I left it as non procedural.

### CODE REVIEW

**Review by MaxwellBo — APPROVED** [review]

**Comment by MaxwellBo** [comment]
LGTM!

File: src/viewmodels/motion.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import { Console } from "console";
```
Comment (MaxwellBo) [inline src/viewmodels/motion.ts:1]:
```suggestion
import { Console } from "console";
```

Actually one quick alteration, can you remove this line

Comment (JulianWww) [reply src/viewmodels/motion.ts]: Done

**Comment by JulianWww** [comment]
Is there anything I have to do to resolve "Missing successful muncoordinated and muncoordinated-staging deployments."?

**Comment by MaxwellBo** [comment]
@JulianWww The deployment pipelines seems to be broken - I will fix that.

Everything seems to be good here!

**Comment by MaxwellBo** [comment]
@JulianWww 

I've fixed Netlify testing on your forks:

Screen Shot 2022-11-28 at 1 17 42 pm

**Pull request merged**